postgres/server/init.sls

```diff
@@ -184,9 +184,6 @@ postgresql-running:
   service.running:
     - name: {{ postgres.service }}
     - enable: True
-   {% if grains.os not in ('MacOS',) %}
-    - reload: True
-   {% endif %}
```

Comment:
@RobRuana This has already been resolved by 79ab1a2 during #218:

https://github.com/saltstack-formulas/postgres-formula/blob/332fe7b2b57f8a1096e91f5920acdea9fd2192e8/postgres/server/init.sls#L141-L151

Reply:
Thanks for the notice! I'm scrambling to get everything working at the moment, and this was my quick and dirty fix. I've synced with upstream.

Reply:
@RobRuana You're welcome.